src/ethereum/ethEngine.ts

```diff
@@ -1169,6 +1174,116 @@ export class EthereumEngine extends CurrencyEngine<EthereumTools> {
     return edgeTransaction
   }
 
+  async accelerate(
+    edgeTransaction: EdgeTransaction
+  ): Promise<EdgeTransaction | null> {
+    const { currencyCode } = edgeTransaction
+
+    const txOtherParams = asMaybe(asEthereumTxOtherParams)(
+      edgeTransaction.otherParams
+    )
+
+    let replacedTxid = edgeTransaction.txid
+    let replacedTxIndex = await this.findTransaction(
+      currencyCode,
+      normalizeAddress(replacedTxid)
+    )
+    if (replacedTxIndex === -1) {
+      if (txOtherParams?.rbfTxid != null && txOtherParams.rbfTxid !== '') {
+        // If the tx parameter is not found, then perhaps it is a
+        // replacement transaction itself
+        replacedTxid = txOtherParams.rbfTxid
+        replacedTxIndex = await this.findTransaction(
+          currencyCode,
+          normalizeAddress(replacedTxid)
+        )
+      }
+
+      if (replacedTxIndex === -1) {
+        // Cannot allow an unsaved (unobserved) transaction to be replaced
+        return null
+      }
+    }
+    const replacedTx: EdgeTransaction =
+      this.transactionList[currencyCode][replacedTxIndex]
+
+    const replacedTxOtherParams = asMaybe(asEthereumTxOtherParams)(
+      replacedTx.otherParams
+    )
+
+    // Transaction checks
+    if (replacedTx == null || replacedTx.blockHeight > 0) {
+      return null
+    }
+    // Other params checks
+    if (
+      replacedTxOtherParams == null ||
+      replacedTxOtherParams.nonceUsed == null ||
+      replacedTxOtherParams.data != null
+    ) {
+      return null
+    }
```

Comment:
Can't we extract the nonce and the data from the raw on-chain transaction?

Edit: Yes, we can use @ethereumjs/tx to do this. However, it doesn't look like this plugin stores the raw on-chain transaction anywhere to begin with. So, maybe we could update `processEvmScanTransaction` and friends to stick the data and nonce into `otherParams`, but for now we rely on our `makeSpend` to do this, and pray the user doesn't do a resync.